src/main/java/org/helpberkeley/memberdata/ControlBlock.java

```diff
@@ -34,6 +37,8 @@ public abstract class ControlBlock {
     static final String ERROR_MISSING_OPS_MANAGER =
             "Control block missing a " + Constants.CONTROL_BLOCK_OPS_MANAGER_USERNAME_ONLY + " entry.\n";
 
+    static final String UNSUPPORTED_VERSION_GENERIC = "Control block version {0} is not supported.\n";
```

Comment:
As part of adding a constant like this, I check the entire project for places where it might apply.  You can use the find command within Intellij or various command line tools .  For example, this should applied at line 146 in this file.